book/analytics/mobile_web_support/mobile_web_support.md

```diff
@@ -66,6 +66,6 @@ To test your work on Android:
 
 1. Integrate your web product in a Zappified Applicaster test app. Contact support at support@applicaster.com if you have questions on how to do this.
 2. Configure analytics for the test app via Zapp. You can find directions for how to configure analytics in Zapp [here](https://applicaster.zendesk.com/hc/en-us/articles/206419186).
-    * Most analytics providers (Flurry, Google Analytics, Facebook Analytics, etc.) do not enable you to send PII (Personally Identifiable Information) or profile-level user data. Applicaster will still send events to these providers. However, for user profiles, Applicaster supports [Mixpanel](https://mixpanel.com/), an analytics tool which both supports the delivery of PII data and provides engagement and remarketing tools based on this data. If you want to test the delivery of user profiles, make sure to configure a test project for Mixpanel in Zapp. 
+    * Most analytics providers (Firebase Analytics, Facebook Analytics, etc.) do not enable you to send PII (Personally Identifiable Information) or profile-level user data. Applicaster will still send events to these providers. However, for user profiles, Applicaster supports [Mixpanel](https://mixpanel.com/), an analytics tool which both supports the delivery of PII data and provides engagement and remarketing tools based on this data. If you want to test the delivery of user profiles, make sure to configure a test project for Mixpanel in Zapp. 

```

Comment:
"If you want to test the delivery of user profiles which includes PII, make sure to configure a test project for Mixpanel or another provider who supports PII delivery in Zapp."